scan.py

```diff
@@ -22,7 +22,7 @@
 import boto3
 
 import clamav
-import metrics
+import
```

Comment:
is the empty `import` intentional?

Reply:
You got there first!

Reply:
It wasn't, it was an error, but fixed before I merged.